backend/oas/provider/openapi.yaml

```diff
@@ -445,6 +458,107 @@ components:
           description: The reason indicating why there is no result
       required:
         - desc
+    jobs.GetJobsResponse:
+      type: object
+      properties:
+        job_id:
+          $ref: '#/components/schemas/jobs.JobId'
+        name:
+          type: string
+          example: Bell State Sampling
+        description:
+          type: string
+          example: Bell State Sampling Example
+        device_id:
+          type: string
+          example: Kawasaki
+        shots:
+          type: integer
+          minimum: 1
+          maximum: 10000000
+          example: '1000'
+        job_info:
+          $ref: '#/components/schemas/jobs.JobInfo'
+        transpiler_info:
+          type: string
+          example: |-
+            {
+              "qubit_allocation": {
+                "0"": 12,
+                "1": 16
+              },
+              "skip_transpilation": false,
+              "seed_transpilation": 873
+            }
+        simulator_info:
+          type: string
+          example: |-
+            {
+              "n_qubits": 5,
+              "n_nodes": 12,
+              "n_per_node": 2,
+              "seed_simulation": 39058567,
+              "simulation_opt": {
+                "optimization_method": "light",
+                "optimization_block_size": 1,
+                "optimization_swap_level": 1
+              }
+            }
+        mitigation_info:
+          type: string
+          example: |
+            { "ro_error_mitigation": "pseudo_inverse" }
+        status:
+          $ref: '#/components/schemas/jobs.JobStatus'
+        created_at:
+          type: string
+          format: date-time
+          example: '2022-10-19T11:45:34+09:00'
+        updated_at:
+          type: string
+          format: date-time
+          example: '2022-10-19T11:45:34'
```

Comment:
`date-time` should be include timezone
'2022-10-19T11:45:34+09:00'

Reply:
https://github.com/oqtopus-team/oqtopus-cloud/blob/8a139a5532484118c3a46561da48ecf3084cbd34/backend/oas/provider/openapi.yaml#L517-L520

fix datetime format